sample.tf

```diff
@@ -370,7 +370,7 @@ resource "artifactory_remote_maven_repository" "maven-remote" {
   fetch_sources_eagerly              = false
   suppress_pom_consistency_checks    = false
   reject_invalid_jars                = true
-  metadata_retrieval_timeout_seconds = 120
+  metadata_retrieval_timeout_secs = 120
```

Comment:
Is this necessary for this PR?

Reply:
I'd like to have it fixed rather sooner than later. It causes an error because the attribute doesn't exist.